src/main/java/ua/danit/final_project/services/WorkCommentServiceImpl.java

```diff
@@ -0,0 +1,80 @@
+package ua.danit.final_project.services;
+
+import org.joda.time.DateTime;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ua.danit.final_project.configuration.StaticCollection;
+import ua.danit.final_project.entities.ShiftComment;
+import ua.danit.final_project.entities.User;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.entities.User'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWPqbco7khxiKmGaO50&open=AWWPqbco7khxiKmGaO50)

[ISSUE_KEY]: AWWPqbco7khxiKmGaO50